charts/timescaledb-multinode/README.md

```diff
@@ -38,6 +38,20 @@ Alternatively, a YAML file that specifies the values for the parameters can be p
 ```console
 helm upgrade --install my-release -f myvalues.yaml .
 ```
+### Secret override
+
+Instead of setting secrets in values.yaml, they can be manually generated.  The following  example is for  timescaledb-access endpoint and timescaledb-data endpoint.
```

Comment:
too many spaces in some places, and I think using the word `endpoint` might be confusing here
need to use the right terminology, or otherwise rephrasing it differently